src/extension.ts

```diff
@@ -248,7 +248,12 @@ export async function activate(context: ExtensionContext) {
         commands.registerCommand('r.runFromLineToEnd', runFromLineToEnd),
         commands.registerCommand('r.runSelectionRetainCursor', runSelectionRetainCursor),
         commands.registerCommand('r.runCurrentChunk', runCurrentChunk),
+        commands.registerCommand('r.runPreviousChunk', runPreviousChunk),
+        commands.registerCommand('r.runNextChunk', runNextChunk),
         commands.registerCommand('r.runAboveChunks', runAboveChunks),
+        commands.registerCommand('r.runFromCurrentToBelowChunks', runFromCurrentToBelowChunks),
```

Comment:
Might be better named `r.runCurrentAndBelowChunks`?

Reply:
Haha definitely agree! `runFromCurrentToBelowChunks` is too long. I'll change it.